e2e/tests/conceptsets/conceptset-edit-rename.spec.ts

```diff
@@ -59,7 +59,7 @@ describe('Editing and rename Concept Set', () => {
     criteriaSearch = await procedures.clickSelectConceptButton();
 
     // Search in Procedures domain
-    procedureName = 'Screening mammography';
+    procedureName = 'cancer screening';
```

Comment:
The cdr bump in #7256 caused no results for 'Screening mammography' in procedures. Updated the search term so the test should pass.